src/integrations/theme/getTheme.ts

```diff
@@ -71,6 +71,11 @@ export async function getTheme() {
 			parsed = mergeJson(parsed, includeTheme)
 		}
 
+		// Push the merged, parsed colors to the global configuration so the variables are available in the webview
```

Comment:
can you pls explain why we need to do this?

Reply:
Maybe it's just a local development issue but the variables aren't making it to the webview without it. Try modifying one of these colors and see if it changes in the your webview without the update

Reply:
i think we can just remove this and only leave the theme json changes in this PR